ratis-proto/src/main/proto/Raft.proto

```diff
@@ -392,6 +392,16 @@ message TransferLeadershipRequestProto {
   RaftPeerProto newLeader = 2;
 }
 
+message TimeoutNowRequestProto {
+  RaftRpcRequestProto serverRequest = 1;
+  uint64 leaderTerm = 2;
```

Comment:
Could we remove this?  leaderLastEntry already has the term.

Reply:
Yes, we can remove this.